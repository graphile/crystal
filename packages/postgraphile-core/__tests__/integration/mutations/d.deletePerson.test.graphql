#> schemas: "d"

mutation {
  deletePersonById(input:{
    id: 1
  }) {
    person{
      id
      firstName
      lastName
      colNoCreate
      colNoUpdate
      colNoOrder
      colNoFilter
      colNoCreateUpdate
      colNoCreateUpdateOrderFilter
    }
  }
}
