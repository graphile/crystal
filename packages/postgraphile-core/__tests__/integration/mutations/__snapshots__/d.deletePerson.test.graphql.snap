// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`execute 1`] = `
Object {
  "data": Object {
    "deletePersonById": Object {
      "person": Object {
        "colNoCreate": "col_no_create1",
        "colNoCreateUpdate": "col_no_create_update1",
        "colNoCreateUpdateOrderFilter": "col_no_create_update_order_filter1",
        "colNoFilter": "col_no_filter1",
        "colNoOrder": "col_no_order1",
        "colNoUpdate": "col_no_update1",
        "firstName": "John",
        "id": 1,
        "lastName": "Smith",
      },
    },
  },
}
`;
